suite-native/module-send/src/components/SendFeesForm.tsx

```diff
@@ -129,6 +131,11 @@ export const SendFeesForm = ({ accountKey, tokenContract }: SendFormProps) => {
         transaction: selectedFeeLevelTransaction,
     });
 
+    const requestDelayedNavigationToOutputsReview = useDelayedNavigationToOutputsReview({
```

Comment:
nit: the hook name should be same as the returned value
```suggestion
    const requestDelayedNavigationToOutputsReview = useRequestDelayedNavigationToOutputsReview({
```